Dockerfile

```diff
@@ -10,9 +10,9 @@ WORKDIR /stage/
 
 COPY requirements.txt .
 RUN pip install --upgrade pip setuptools wheel
-RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
+RUN pip install torch==2.0.1 torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
```

Comment:
is it possible to bump to 2.1.0 at some point?

Reply:
Some of our dependencies require 2.0.1 iirc. Currenly, 2.1.0 might be too new to incorporate unless there are some features we need. Otherwise, let's revisit later.